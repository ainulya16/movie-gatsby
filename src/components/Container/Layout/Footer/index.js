import React from 'react';
import { Layout } from 'antd/lib';

import styled from 'styled-components';
import QuickSection from '../../../QuickSection';

const DefaultWrapper = styled.div`
  margin-top: 50px;
  .hidenSP {
    display: inline-block;
  }
  .hiddenPC {
    display: none;
  }
  @media (max-width: 767px) {
    .hiddenSP {
      display: none;
    }
    .hiddenPC {
      display: inline-block;
    }
  }
`;
const { Footer: AntdFooter } = Layout;

const children = {
 copyright : 'AIA Â©2019 Created by AIA Teams',
 urlTermsOfUse : '/PA100/terms-of-use'
}

const Footer = () => (
  <DefaultWrapper>
    <AntdFooter className="text-center">{children}</AntdFooter>
    <QuickSection className="quick-section" />
  </DefaultWrapper>
);

export default Footer;
