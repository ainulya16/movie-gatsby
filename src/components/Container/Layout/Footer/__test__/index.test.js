/* eslint-env jest */
import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Footer from '../index';

describe('Footer', () => {
  it('should be exists', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper.exists()).toBe(true);
  });
  it('should match the snapshot', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
