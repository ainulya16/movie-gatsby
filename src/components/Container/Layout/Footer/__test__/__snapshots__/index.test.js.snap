// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot 1`] = `
.c2 {
  min-width: 250px;
  width: 250px;
  text-align: center;
}

.c3 {
  height: 45px;
  font-family: 'AIAMedium';
  text-transform: none;
  font-size: 14px;
  border: 0px;
  display: inline-block;
  background: #D31145;
  -webkit-transition: width 0.3s ease,min-width 0.3s ease,background-color 0.3s ease,border-radius 0.3s ease,border 0.3s ease,display 0.3s ease;
  transition: width 0.3s ease,min-width 0.3s ease,background-color 0.3s ease,border-radius 0.3s ease,border 0.3s ease,display 0.3s ease;
  color: primary;
  background-color: #FFF;
  background-image: linear-gradient(to bottom,#d31145,#D31145,#d31145);
  border: none;
}

.c3:active {
  -webkit-transform: scale(0.9);
  -ms-transform: scale(0.9);
  transform: scale(0.9);
  -webkit-transition: scale 0.1s ease-in-out;
  transition: scale 0.1s ease-in-out;
}

.c3:focus {
  outline: none;
}

.c3:disabled {
  -webkit-transform: scale(1);
  -ms-transform: scale(1);
  transform: scale(1);
  opacity: 0.4;
  -webkit-filter: alpha(opacity=40);
  filter: alpha(opacity=40);
}

.c1 {
  position: -webkit-sticky;
  position: sticky;
  bottom: 0px;
}

.c0 {
  margin-top: 50px;
}

.c0 .hidenSP {
  display: inline-block;
}

.c0 .hiddenPC {
  display: none;
}

@media (max-width:767px) {
  .c0 .hiddenSP {
    display: none;
  }

  .c0 .hiddenPC {
    display: inline-block;
  }
}

<div
  className="c0"
>
  <footer
    className="text-center ant-layout-footer"
  >
    AIA Â©2019 Created by AIA Teams
  </footer>
  <div
    className="quick-section c1"
    style={
      Object {
        "position": "fixed",
        "width": "100vw",
      }
    }
  >
    <div
      className="ant-row"
    >
      <div
        className="ant-col ant-col-xs-6 ant-col-sm-6"
      >
        <div
          className="c2"
          style={Object {}}
        >
          <button
            className="btn  c3"
            color="primary"
            disabled={false}
            loading={false}
            radius={null}
          >
            <div
              className="child"
            >
              Cart
            </div>
          </button>
        </div>
      </div>
      <div
        className="ant-col ant-col-xs-6 ant-col-sm-6"
      >
        <div
          className="c2"
          style={Object {}}
        >
          <button
            className="btn  c3"
            color="primary"
            disabled={false}
            loading={false}
            radius={null}
          >
            <div
              className="child"
            >
              Share
            </div>
          </button>
        </div>
      </div>
      <div
        className="ant-col ant-col-xs-6 ant-col-sm-6"
      >
        <div
          className="c2"
          style={Object {}}
        >
          <button
            className="btn  c3"
            color="primary"
            disabled={false}
            loading={false}
            radius={null}
          >
            <div
              className="child"
            >
              Chat
            </div>
          </button>
        </div>
      </div>
      <div
        className="ant-col ant-col-xs-6 ant-col-sm-6"
      >
        <div
          className="c2"
          style={Object {}}
        >
          <button
            className="btn  c3"
            color="primary"
            disabled={false}
            loading={false}
            radius={null}
          >
            <div
              className="child"
            >
              Advisor
            </div>
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
